src/twisted/mail/test/test_pop3.py

```diff
@@ -496,36 +672,96 @@ def test_badUTF8CharactersInCommand(self):
 
 @implementer(pop3.IServerFactory)
 class TestServerFactory:
+    """
+    A L{pop3.IServerFactory} implementation with some hard-coded and
+    easily-controlled values for use by the test suite.
+    """
     def cap_IMPLEMENTATION(self):
+        """
+        Return the hard-coded value.
+
+        @return: L{pop3.IServerFactory}
+        """
         return "Test Implementation String"
 
 
     def cap_EXPIRE(self):
+        """
+        Return the hard-coded value.
+
+        @return: L{pop3.IServerFactory}
+        """
         return 60
 
     challengers = OrderedDict([(b"SCHEME_1", None), (b"SCHEME_2", None)])
 
     def cap_LOGIN_DELAY(self):
+        """
+        Return the hard-coded value.
+
+        @return: L{pop3.IServerFactory}
+        """
         return 120
 
     pue = True
     def perUserExpiration(self):
+        """
+        Return the hard-coded value.
+
+        @return: L{pop3.IServerFactory}
+        """
         return self.pue
 
     puld = True
     def perUserLoginDelay(self):
+        """
+        Return the hard-coded value.
+
+        @return: L{pop3.IServerFactory}
+        """
         return self.puld
 
 
 
 class TestMailbox:
+    """
+    An incomplete L{IMailbox} implementation with certain well-known values
```

Comment:
Tests hard code the values of [`loginDelay`](https://github.com/twisted/twisted/blob/9310-pop3-twistedchecker/src/twisted/mail/test/test_pop3.py#L842) and [`messageExpiration`](https://github.com/twisted/twisted/blob/cdba3cf8eb8b4bd1585b7339dd40cc65292ab60a/src/twisted/mail/test/test_pop3.py#L811).  Is that why these values are well-known?  If so, "with certain values that appear in this module's tests" would be clearer.